Update CLI install commands (#27859)
dotnet/docs: docs/machine-learning/how-to-guides/install-ml-net-cli.md

## Pre-requisites

> - [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)

Reviewer: ```suggestion
- [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
```

- (Optional) [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)

You can run the generated C# code projects with Visual Studio by pressing the `F5` key or with `dotnet run` (.NET CLI).

Note: If after installing .NET SDK the `dotnet tool` command is not working, sign out from Windows and sign in again.

## Install

The ML.NET CLI is installed like any other dotnet Global Tool. You use the `dotnet tool install --global` .NET CLI command.

> The ML.NET CLI is available for Windows, macOS, and Linux. There are also different versions of the CLI depending if you have Intel based processor or an ARM based processor.

Reviewer: ```suggestion
The ML.NET CLI is available for Windows, macOS, and Linux. There are also different versions of the CLI for x64 and ARM based processors.
```

> [!NOTE]
> Installations with the ARM based versions of the ML.NET CLI won't have access to the image classification task due to limitations on the processor.

# [Windows](#tab/windows)

> ## Intel

Reviewer: I think we can update Intel with x64.
```suggestion
## x64
```

## Pre-requisites

> - [.NET 5](https://dotnet.microsoft.com/download/dotnet/5.0)

Reviewer 2: ```suggestion
- [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
```

The ML.NET CLI is installed like any other dotnet Global Tool. You use the `dotnet tool install` .NET CLI command.

The ML.NET CLI is available for Windows, macOS, and Linux.

# [Windows](#tab/windows)

> ```console

Reviewer 2: ~~~suggestion
```dotnetcli
~~~

dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.Win64
```

# [Mac](#tab/mac)

> ```console

Reviewer 2: ~~~suggestion
```dotnetcli
~~~

dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.OSX
```

# [Linux](#tab/linux)

> ```console

Reviewer 2: ~~~suggestion
```dotnetcli
~~~

dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.Linux64
```

> 

Reviewer 2: ```suggestion

---

```
